Binary Tree Traversal/preorder-inorder-postorder.c

```diff
@@ -0,0 +1,268 @@
+#include<stdio.h>
+#include<malloc.h>
+#include<stdlib.h>
+struct node
+{
+int data;#include<stdio.h>

```

Comment:
* Please check your code.You have added `#include<stdio.h>`.
also close properly close the `struct node`
 * Please add README.md file inside `Binary Tree Traversal` directory.write some description about algorithm & input & output sample.